plugin_1_4.yaml

```diff
@@ -3,11 +3,34 @@ plugins:
   ansible:
     executor: central_deployment_agent
     package_name: cloudify-ansible-plugin
-    package_version: '2.13.17'
+    package_version: '3.0.0'
 
 dsl_definitions:
 
+  ansible_install_config: &install_config
+    use_existing_resource:
+      type: boolean
+      description: If true, use an existing ansible installation rather than installing it
+      default: false
+    installation_source:
+      type: string
+      default: 'https://cloudify-release-eu.s3.eu-west-1.amazonaws.com/cloudify/wagons/cloudify-ansible-plugin/ansible-2.9.5-py3-none-any.whl'
```

Comment:
@szpotona let's match the plugin yaml values [ those are different than plugin.yaml ? ]